tests/unit/packet/PublishTest.php

```diff
@@ -225,65 +244,67 @@ public function testParseWithQos($qos, $bit)
     {
         $input =
             chr($bit) .
-            chr(4) .
+//            chr(4) .
```

Comment:
Ok, so I think the numbers don't mean anything. As long as that message has at least 4 bytes of data, you can fill it up with any `null` characters.

You can change the original test and it will still pass:

```
            chr(56) .
            chr(4) .
            chr(0) .
            chr(0) .
            chr(0) .
            chr(0) .
            chr(0) .
            chr(0) .
            chr(10);
```
or
```
            chr(56) .
            chr(4) .
            chr(0) .
            chr(0);